docs/RELEASE.md

```diff
@@ -0,0 +1,126 @@
+# Release
+
+The output of a release are the `polkadot` node, runtimes for the Westend & Rococo networks and new crate versions published to `crates.io`.
+
+## Versioning
+
+We are releasing multiple different things from this repository in one release, but 
+we don't want to use the same version for everything. Thus, in the following we explain
+the versioning story for the crates, node and Westend & Rococo. To easily refer to a release, we shall use the node version of it.
+
+### Crate
+
+We try to follow SemVer<sup>3</sup> as best as possible for versioning our crates' public APIs.  
+
+👉 The public API of our library crates is defined as all public items that are not inside a `__private` module.
+
+### Node
+
+The versioning of the node is done most of the time by only incrementing the `minor` version. 
+The `major` version is only bumped for special releases and the `patch` can be used for an 
+out of band release that fixes some critical bug. The node version is not following SemVer. 
+This means that the version doesn't express if there are any breaking changes in the CLI 
+interface or similar. The node version is declared in the `NODE_VERSION` variable in 
+`polkadot/node/primitives/src/lib.rs`.
+
+### Westend & Rococo
+
+For the these networks, in addition to incrementing the Cargo.toml version we also increment the `spec_version` and sometimes the `transaction_version`. The spec version is also following
+the node version. Its schema is: `M_mmm_ppp` and for example `1_002_000` is the node release `1.2.0`. This versioning has no further meaning, and is only done to map from an on chain `spec_version` easily to the release in this repository. 
+
+## Backports
+
+Backports should be rare. We should only backport [critical bug fixes](#bug-and-security-fix) and then release the fixed crates. There should be no need to backport anything from a release branch.
```

Comment:
What about new features or performance improvements? Not having any progress for 3 months is quite long. I mean as long as these new things to do not break APIs or whatever.

Reply:
Yeah if there's something we think would be useful to get out earlier makes sense to add it to both `stable` and `master`.

Reply:
We can also backport features/improvements, but only after they were audited. I will add a section for it.